source/includes/swift/_objects.md

```diff
@@ -0,0 +1,262 @@
+   -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/v1/services/swift/example/objects?bucket_name=private"
+
+# The above command returns JSON structured like this:
+```
+
+```json
+{
+  "data": [
+    {
+      "id": "private/myfile_7g.bin-dlo",
+      "name": "myfile_7g.bin-dlo",
+      "displayName": "myfile_7g.bin-dlo",
+      "path": "",
+      "size": 7073741824,
+      "isNode": false,
+      "bucketName": "private",
+      "displaySize": {
+        "value": "6.6",
+        "unitKey": "units.filesizes.GB"
+      },
+      "contentType": "application/octet-stream",
+      "lastModified": "Wed Mar 11 16:51:20 EDT 2020"
+    },
+    {
+      "id": "private/exampleFolder",
+      "name": "exampleFolder",
+      "displayName": "exampleFolder",
+      "path": "",
+      "size": 0,
+      "isNode": true,
+      "bucketName": "private",
+      "contentType": "application/directory",
+      "lastModified": "Tue Mar 10 17:55:28 EDT 2020"
+    },
+    {
+      "id": "private/exampleFolder/sampleFile.txt",
+      "name": "sampleFile.txt",
+      "displayName": "sampleFile.txt",
+      "path": "",
+      "size": 27,
+      "isNode": false,
+      "bucketName": "private",
+      "displaySize": {
+        "value": "27",
+        "unitKey": "units.filesizes.B"
+      },
+      "contentType": "application/octet-stream",
+      "lastModified": "Wed Mar 11 16:51:20 EDT 2020"
+    }
+  ],
+  "metadata": {
+    "recordCount": 3
+  }
+}
+```
+
+<code>GET /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/objects?bucket_name=:bucket_name</code>
+
+Retrieve a list of all objects in a given [environment](#administration-environments) in the specified container.
+
+Mandatory | &nbsp;
+------- | -----------
+`bucket_name` <br/>*string* | The name of the bucket from which the object must be fetched.
+
+Optional | &nbsp;
+------- | -----------
+`path` <br/>*string* | Limits the object list to the provided path. (Simulating a bit a folder structure). It does not list "sub-folders".
+`show_all` <br/>*boolean* | Combined with the path, it will return also the "sub-folders".
+`q` <br/>*string* | Filters the object with the name contains the value of the parameters.
+
+Attributes | &nbsp;
+------- | -----------
+`id` <br/>*string* | The id of the object .
+`name` <br/>*string* | The name of the object.
+`displayName`<br/>*string* | The name rendered for display.
+`path`<br/>*string* | The path of the object.
+`size`<br/>*int* | The size in bytes of the object. It is 0 for object that are used as folders (Generated by UI).
+`isNode`<br/>*boolean* | The access type of the container. It can either be public or private.
+`bucketName`<br/>*string* | The name of the bucket that the object is part of.
+`displaySize`<br/>*object* | Object containing the information to display the size in the UI.
+`contentType`<br/>*string* | The mime type corresponding to the content of the file.
```

Comment:
Can you capitalize MIME?

Reply:
cdaa1dc4734e0146a9f3a346d80e276c4993bfbe